autotest/gdrivers/nitf.py

```diff
@@ -3613,7 +3613,7 @@ def test_nitf_80():
 
 
 def test_nitf_81():
-    tre_data = "TRE=MSTGTA=012340123456789AB0123456789ABCDE0120123456789AB0123456789AB000123401234560123450TGT_LOC=             "
+    tre_data = "TRE=MSTGTA=012340123456789AB0123456789ABCDE0120123456789AB0123456789AB001234501234560123450TGT_LOC=             "
```

Comment:
I don't see the difference!

Reply:
there's one at column 89: "01234" -> "12345". Was needed to meet a min/max check